### PR TITLE
test: [SFEQS-1552] Add unit test create signature request

### PR DESCRIPTION
**Pull request by silvicir**
#### List of Changes
1. Migrated `CreateSignatureRequest` in `io-func-sign-issuer` workspace to `handler-kit@1`
2. Added unit test
3. Updated `openapi.yaml` to return `422` if body is invalid in `/signature-requests POST`

#### How Has This Been Tested?

Unit test + manual e2e test in development environment

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by gunzip — COMMENTED** [review]
very good!!!

File: apps/io-func-sign-issuer/src/infra/http/handlers/create-signature-request.ts [hunk 1]
```diff
@@ -68,6 +69,9 @@ export const CreateSignatureRequestHandler = H.of((req: H.HttpRequest) =>
       )
     ),
     RTE.chainW(insertSignatureRequest),
+    RTE.chainFirstW((request) =>
+      pipe(EventName.SIGNATURE_CREATED, createAndSendAnalyticsEvent(request))
+    ),
```
Comment (gunzip) [inline apps/io-func-sign-issuer/src/infra/http/handlers/create-signature-request.ts:74]: I suspect that the order of arguments for `createAndSendAnalyticsEvent` should be swapped. Usually the principle is to put configuration first (`EventName.SIGNATURE_CREATED`) and data (`request`) last. In this case you could have used:

```suggestion
    RTE.chainFirstW(createAndSendAnalyticsEvent(EventName.SIGNATURE_CREATED)),
```

Comment (gunzip) [reply apps/io-func-sign-issuer/src/infra/http/handlers/create-signature-request.ts:74]: see also https://stackoverflow.com/questions/5863128/ordering-of-parameters-to-make-use-of-currying

Comment (silvicir) [reply apps/io-func-sign-issuer/src/infra/http/handlers/create-signature-request.ts:74]: I had already discussed this @lucacavallaro. We decided to take a look at the function `createAndSendAnalyticsEvent` at a later time with @grausof.

Comment (grausof) [reply apps/io-func-sign-issuer/src/infra/http/handlers/create-signature-request.ts:74]: Right! I would say make this change with a separate PR.

**Review by grausof — APPROVED** [review]
LGTM

**Pull request merged**